.gitignore

```diff
@@ -31,7 +31,7 @@ pyrightconfig.json
 doc/_build/
 *.swp
 .DS_Store
-
+readme_misc.md
```

Comment:
What is this?

Reply:
This was my read_me with grammatical mistakes. I will remove it from .gitignore.